src/components/TicTacToe.tsx

```
@@ -0,0 +1,98 @@
+    setBoard(Array(9).fill(null));
+    setCurrentPlayer('X');
+    setWinner(null);
+  };
+
+  const renderSquare = (index: number) => {
+    return (
+      <button
+        className={styles.square}
+        onClick={() => handleClick(index)}
+        disabled={!!winner || !!board[index]}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)